ipaserver/install/server/replicainstall.py

```diff
@@ -747,6 +786,10 @@ def install_check(installer):
                          ldap_uri=ldapuri)
     remote_api.finalize()
     installer._remote_api = remote_api
+
+    with rpc_client(remote_api) as client:
+        check_remote_fips_mode(client, api.env.fips_mode)
```

Comment:
Why are you checking fips mode twice in DL1? Also, if this is supposed to address the issue "Explode if trying to install in FIPS on DL0", this won't work.